### PR TITLE
Update repository

### PR DESCRIPTION
**Pull request by DavideD**
Update a repository that has been deprecated

### CODE REVIEW

File: hibernate-orm/core/Spatial/Spatial.gradle [hunk 1]
```diff
@@ -7,12 +7,12 @@ repositories {
         url "http://www.hibernatespatial.org/repository"
     }
     maven {
-        url "http://repo.opengeo.org"
+        url "http://repo.boundlessgeo.com/main"
     }
 }
 
 dependencies {
 	compile( 'org.hibernate:hibernate-spatial:4.3-SNAPSHOT')
```
Comment (gsmet) [inline hibernate-orm/core/Spatial/Spatial.gradle]: 4.3-SNAPSHOT?

As you volunteered, you should probably upgrade to a more recent version :).

Comment (DavideD) [reply hibernate-orm/core/Spatial/Spatial.gradle]: I don't know if I want to go so deep into the demo.
I'll update to a 4.3   Final for now

Comment (gsmet) [reply hibernate-orm/core/Spatial/Spatial.gradle]: Well, let's merge this one and let the ORM guys update the demos :).

**Comment by gsmet** [comment]
Merged, thanks Davide.

**Comment by DavideD** [comment]
Thanks @gsmet,
just as a note, now spatial is included in hibernate ORM so, to update the issue, I don't think it's enough to upgrade the version

**Comment by gsmet** [comment]
Yeah, it would be nice if someone in the ORM team could find some time to update these demos. People are looking at them.

**Pull request closed without merging**